jenkins/ubuntu/docker/Dockerfile

```diff
@@ -38,6 +38,8 @@ RUN echo 'BatchMode yes' >>/etc/ssh/ssh_config && \
 RUN usermod -a -G sudo jenkins && \
     usermod -a -G adm  jenkins
 
+RUN echo "deb http://apt.openvstorage.org unstable main" > /etc/apt/sources.list.d/ovsaptrepo.list
```

Comment:
I'd rather put this in the build_debs.sh script instead with a remark that some dependencies for the volumedriver are only available there. 

Note: build_debs.sh doesn't run as root so a sudo is needed (eg. echo 'deb http://apt.openvstorage.org unstable main' | sudo tee /etc/apt/sources.list.d/ovsaptrepo.list)
 
Hint: the same build_debs.sh script is present in the qemu & blktap repo